### PR TITLE
Update GPU performance and neuron_menu

### PR DESCRIPTION
**Pull request by DanielHo-BS**
1. Update the content of GPU card:
  i. roscube-i/gpu_card/index.rst
  ii. common/gpu_performance/ai_benchmark.rst

2. Add the usage of AI Benchmark:
  i. common/gpu_performance/index.rst
  ii. common/gpu_performance/ai_benchmark_usage.rst

3.Add [docker.io](http://docker.io/) for ros_menu installation

### CODE REVIEW

File: roscube-i/gpu_card/index.rst [hunk 1]
```diff
@@ -10,11 +10,34 @@ In this chapter, we show you:
...
 
-These GPU cards which be used on **ROScube-I-E** :
+**GPU Table:**
+
+These GPU cards which be used on **ROScube-I-E**.
+
+**1. ADLINK:**
+
+    * Quadro PEG T1000 
+    * Quadro-E PEG P1000
```
Comment (evshary) [inline roscube-i/gpu_card/index.rst]: I think we don't need to list the GPU model which will EOL soon.

File: roscube-i/gpu_card/index.rst [hunk 2]
```diff
@@ -10,11 +10,34 @@ In this chapter, we show you:
...
+
+.. note:: 
+
+    For RQI series, we need to use ATX SKU.
+
+.. note::
```
Comment (evshary) [inline roscube-i/gpu_card/index.rst]: We don't need to list PN number here.

File: roscube-i/gpu_card/index.rst [hunk 3]
```diff
@@ -10,11 +10,34 @@ In this chapter, we show you:
...
+    | P1000: 91-95259-0010 (ATX bracket). Will EOL pretty soon.
+
+**2. Others:**
 
     * NVIDIA RTX A2000  12G
     * NVIDIA GTX 1050   2G
 
+.. warning:: 
+    
+    In others, we've not tested these GPU cards under critical thermal & vibration condition.
```
Comment (evshary) [inline roscube-i/gpu_card/index.rst]: Both ADLINK and others are not tested under critical thermal & vibration conditions.
So maybe we don't need to separate ADLINK and others.

File: common/gpu_performance/index.rst [hunk 4]
```diff
@@ -11,7 +11,6 @@ The chapter shows the GPU performance on ROScube series.
    :maxdepth: 1
 
    ai_benchmark.rst
```
Comment (evshary) [inline common/gpu_performance/index.rst]: image
The order is suggested like the following
How to do AI benchmark
Benchmark result

File: common/gpu_performance/ai_benchmark_usage.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+In total, AI Benchmark consists of 42 tests and 19 sections.
+
+After testing, you will get the Score of GPU performance:
+
+    * Inference Score
+    * Training Score
+    * AI Score
+
+Then go to the `ranking page <https://ai-benchmark.com/ranking_deeplearning.html>`_, and you can compare your device with open data.
```
Comment (evshary) [inline common/gpu_performance/ai_benchmark_usage.rst:95]: You can mention that users can refer to the benchmark result we've tested.

**Review by evshary — CHANGES_REQUESTED** [review]

File: roscube-i/gpu_card/index.rst (hunk 1 above)
Comment (DanielHo-BS) [reply roscube-i/gpu_card/index.rst]: OK

File: roscube-i/gpu_card/index.rst (hunk 2 above)
Comment (DanielHo-BS) [reply roscube-i/gpu_card/index.rst]: OK

File: roscube-i/gpu_card/index.rst (hunk 3 above)
Comment (DanielHo-BS) [reply roscube-i/gpu_card/index.rst]: OK

File: common/gpu_performance/index.rst (hunk 4 above)
Comment (DanielHo-BS) [reply common/gpu_performance/index.rst]: OK

File: common/gpu_performance/ai_benchmark_usage.rst (hunk 5 above)
Comment (DanielHo-BS) [reply common/gpu_performance/ai_benchmark_usage.rst:95]: I got some warning when I was doing the link.

/home/ros/roscube-doc/nvidia/advanced/common/gpu_performance/ai_benchmark.rst:41: WARNING: duplicate label gpu score, other instance in /home/ros/roscube-doc/common/gpu_performance/ai_benchmark.rst
/home/ros/roscube-doc/roscube-i/gpu_card/common/gpu_performance/ai_benchmark.rst:41: WARNING: duplicate label gpu score, other instance in /home/ros/roscube-doc/nvidia/advanced/common/gpu_performance/ai_benchmark.rst
/home/ros/roscube-doc/roscube-pico/nvidia/advanced/common/gpu_performance/ai_benchmark.rst:41: WARNING: duplicate label gpu score, other instance in /home/ros/roscube-doc/roscube-i/gpu_card/common/gpu_performance/ai_benchmark.rst
/home/ros/roscube-doc/roscube-x/nvidia/advanced/common/gpu_performance/ai_benchmark.rst:41: WARNING: duplicate label gpu score, other instance in /home/ros/roscube-doc/roscube-pico/nvidia/advanced/common/gpu_performance/ai_benchmark.rst


But I think it's fine, it is just duplicate label when we use the shut link folder(common folder).

**Review by DanielHo-BS — COMMENTED** [review]
Done the request changes, but I got some "warning: duplicate label".
I think it fine, but do you have a better way to solve it?

File: roscube-i/gpu_card/index.rst [hunk 6]
```diff
@@ -10,11 +10,21 @@ In this chapter, we show you:
...
 
-These GPU cards which be used on **ROScube-I-E** :
+**GPU Table:**
 
+These GPU cards which be used on **ROScube-I-E**.
+
+
+
+    * ADLINK Quadro PEG T1000 
+    * ADLINK Quadro-E PEG P1000
```
Comment (evshary) [inline roscube-i/gpu_card/index.rst]: P1000 is about to EOL, so we don't need to list it.

File: common/gpu_performance/index.rst [hunk 7]
```diff
@@ -10,8 +10,5 @@ The chapter shows the GPU performance on ROScube series.
 .. toctree::
    :maxdepth: 1
 
-   ai_benchmark.rst
-.. This should be added in the future
    ai_benchmark_usage.rst
-   
-   
\ No newline at end of file
+   ai_benchmark.rst
```
Comment (evshary) [inline common/gpu_performance/index.rst:14]: I think the title here should be "AI Benchmark Result"

File: common/gpu_performance/ai_benchmark_usage.rst (hunk 5 above)
Comment (evshary) [reply common/gpu_performance/ai_benchmark_usage.rst:95]: I think you can use relative path to solve the issue.

**Review by evshary — CHANGES_REQUESTED** [review]

**Review by DanielHo-BS — COMMENTED** [review]
Done the request changes

**Comment by evshary** [comment]
LGTM

**Pull request merged**